scard.go

```diff
@@ -198,7 +198,11 @@ func (card *Card) Status() (*CardStatus, error) {
 		return nil, err
 	}
 
-	reader := decodemstr(strbuf(readerBuf[:readerLen]))
+	sb, e := encodestr(string(readerBuf[:readerLen]))
```

Comment:
I think this breaks when there are multiple reader names (and thus terminating 0s inside the string).